lib/pbench/agent/fs.py

```diff
@@ -0,0 +1,23 @@
+import errno
+import os
+import shutil
+
+
+def safe_rmtree(directory):
+    """Delete a directory, if it's present otherwise no-op"""
+    if os.path.exists(directory):
+        shutil.rmtree(directory)
+
+
+def safe_mkdir(directory, clean=False):
+    """Safely create a directory.
+    Ensures a directory is present.  If it's not there, it is created.  If it is, it's a no-op. If
+    clean is True, ensures the directory is empty.
+    """
+    if clean:
+        safe_rmtree(directory)
```

Comment:
"ensures the directory is empty" ... ha. But I guess at the command level, deleting an empty directory and re-creating it shouldn't be a big deal.

Reply:
Done